src/EventStore.Core/EventStore.Core.csproj

```diff
@@ -22,10 +22,16 @@
 			<IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
 		</PackageReference>
 		<PackageReference Include="HdrHistogram" Version="2.5.0" />
+		<PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0-beta2" />
+		<PackageReference Include="Microsoft.ApplicationInsights.EventCounterCollector" Version="2.22.0-beta2" />
 		<PackageReference Include="Microsoft.FASTER.Core" Version="1.9.5" />
-		<PackageReference Include="OpenTelemetry.Exporter.Prometheus.AspNetCore" Version="1.4.0-rc.1" />
-		<PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.4.0-rc.1" />
+		<PackageReference Include="OpenTelemetry.Exporter.Prometheus.AspNetCore" Version="1.5.0-alpha.1" />
 		<PackageReference Include="Quickenshtein" Version="1.5.1" />
+		<PackageReference Include="OpenTelemetry.Api.ProviderBuilderExtensions" Version="1.5.0-alpha.1" />
+		<PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.5.0-alpha.1" />
```

Comment:
not sure about upgrading from a RC to an alpha 😅  was there something that we needed in the upgraded version?

Reply:
Well the initial worked needed it! Might not be the case anymore